src/app/api/images/copy-reference/route.ts

```
@@ -0,0 +1,199 @@
+      name: string
+      description?: string | null
+      tags?: string | null
+    } | null = null
+
+    switch (sourceType) {
+      case 'main':
+        const [mainImg] = await db
+          .select()
+          .from(imageData)
+          .where(eq(imageData.id, parseInt(imageId)))
+          .limit(1)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)